test/gcs_taskrun_test.go

```diff
@@ -1,106 +0,0 @@
-// +build e2e
-
-/*
-Copyright 2019 The Tekton Authors
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package test
-
-import (
-	"os"
-	"path/filepath"
-	"testing"
-
-	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
-	tb "github.com/tektoncd/pipeline/test/builder"
-	knativetest "knative.dev/pkg/test"
-)
-
-// TestStorageTaskRun is an integration test that will verify GCS input resource runtime contract
-// - adds volumes with expected name
-// - places files in expected place
-
-func TestStorageTaskRun(t *testing.T) {
-	configFilePath := os.Getenv("GCP_SERVICE_ACCOUNT_KEY_PATH")
-	if configFilePath == "" {
-		t.Skip("GCP_SERVICE_ACCOUNT_KEY_PATH variable is not set.")
-	}
```

Comment:
if we can get away with replacing this with an example test you've also tackled half of https://github.com/tektoncd/pipeline/issues/821 !! are you sure the example test has the permissions it needs to pull this off? im just a bit surprised :D

Reply:
The YAML test(s) in this PR pass, because the objects they read from GCS are publicly visible and don't need creds at all to be fetched. :tada:

I'll mark this issue as closing #821

Reply:
Thanks for bringing that issue to my attention. This doesn't actually fix #821 though it helps. I added a comment there with an error message I get, we can address non-`build-gcs` resources in a future change.